vmdb/app/models/miq_request.rb

```diff
@@ -502,4 +472,11 @@ def self.update_request(request, values, requester_id, target_class, event_messa
     return request
   end
 
+  private
+
+  def request_types
```

Comment:
is this called that often?
Is it ok to generate over and over?
What do you think about passing in the template up front?
```ruby
request_types = {:MiqProvisionRequestTemplate => {:template => "VM Provision Template"}}
MODEL_REQUEST_TYPES.values.each_with_object(request_types) { |i, h| i.each { |k, v| h[k] = v } }
```